backend/src/impl/default_controllers_impl.py

```diff
@@ -189,7 +195,7 @@ def systems_system_id_outputs_get(
     """
     TODO: return special error/warning if some ids cannot be found
     """
-    sys = SystemModel.find_one_by_id(system_id)
+    sys = DBUtils.find_one_by_id(DBUtils.DEV_SYSTEM_METADATA, system_id)
```

Comment:
An error is raised in `DatalabLoaderOption(...` because `sys.system_info` is not unbinarized.
I think there are 3 places where we need to find the system by id (two in this file and one in system_db_utils.py). Do you think it would be more convenient if we could have a static method in `SystemDBUtils` to handle that? That method can also call `system_from_dict()` so we always get a `System` object from it.